.jovianrc

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
this needs to be removed. if any test requires a `.jovianrc`, you can create and remove it using testing helpers as done in other tests.